Actions/DumpWorkflowInfo/action.yaml

```diff
@@ -0,0 +1,24 @@
+name: Dump Workflow Info
```

Comment:
It does?! Where? 
I just ran [this one](https://github.com/microsoft/BCApps/actions/runs/7003393027/job/19049038761) and didn't notice the input output anywhere.

Otherwise, yes, that's for the user input when dispatching a workflow.

Reply:
I also didn't find any log of the inputs given to GitHub Actions?
Reason for the name is really that if we find that other info about the workflow would be nice to get in a central place for troubleshooting purposes, this is the place to put it.

Reply:
And I actually like Dump better than Log:-)